package.json

```diff
@@ -20,8 +20,10 @@
     "docs:serve": "mkdocs serve",
     "link:all": "lerna exec yarn link",
     "lint": "eslint --no-eslintrc --c ./.eslintrc.js --cache --report-unused-disable-directives --format codeframe --ext js,jsx \".*.js\" packages",
-    "release": "lerna publish",
-    "release:preview": "lerna publish --skip-git --skip-npm",
+    "release:prepare": "lerna version",
+    "release:publish": "lerna publish from-package",
+    "release:publish-next": "yarn release:publish --dist-tag next",
+    "release:ci": "lerna publish prerelease --registry http://localhost:4873 --no-git-reset --yes",
```

Comment:
```suggestion
    "release:ci": "SKIP_CHANGELOG=true lerna publish --registry http://localhost:4873 --no-git-reset --yes --canary prerelease",
```

Reply:
Re this suggested change - moving the `SKIP_CHANGELOG` here is a good idea, however the `from-package` option conflicts with the other options passed, so doesn't make sense in this context. See:
https://github.com/lerna/lerna/blob/a7ad9b60d27b390fde21fd2837f2d97320c4603e/commands/publish/index.js#L135-L143

Re `--canary`, we used to use that, however it used to append the Git SHA to the version in a way that created a semver-spec-violating version string if the Git SHA happened to have a leading zero, so stopped doing so. However it looks like that was fixed in lerna v3 (lerna/lerna/issues/1118), so we can switch back to using it :-)

Reply:
> However it looks like that was fixed in lerna v3 ([lerna/lerna/issues/1118](https://github.com/lerna/lerna/issues/1118)), so we can switch back to using it :-)

Ah `--canary` ignores the Neutrino version in `lerna.json` and instead tries to base it on the Git tags, which (a) doesn't work well on Travis (see https://travis-ci.org/neutrinojs/neutrino/jobs/507436675#L564), (b) means it uses a v8.x.x version rather than v9.x.x, since we don't yet have any Neutrino v9 Git tags.

Given we're not actually the packages in canary form, we don't benefit much from `--canary`, so I've just switched this back to how it was before.